config/routes.rb

```diff
@@ -7,4 +7,9 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  resources :users, only: [:index, :show] do
+      resources :posts, only: [:index]
```

Comment:
- [ ] The posts' `show` route should also be wrapped within the users route, kindly update the route on line 11.

  ```
  resources :posts, only: [:index, :show]
  ```